include/sys/timer.h

```diff
@@ -42,6 +42,10 @@ typedef struct timer {
   void *tm_priv;              /*!< private data (usually device_t *) */
 } timer_t;
 
+/* Used by rtc to initalize system boottime using rtc */
+void boottime_init(tm_t *t);
+/* Used to set/change system boottime */
+void tm_setclock(const timespec_t *ts);
 /*! \brief Used by a driver to make timer device available to the system. */
```

Comment:
You can consider changing the comments to `/*! \brief ... */` format to be consistent with the rest of the file